### PR TITLE
Add extra support docs for publish sidekiq

### PR DESCRIPTION
**Pull request by kentsanggds**
Add documentation for clearing a sidekiq queue if there is an issue in processing it

### CODE REVIEW

File: source/manual/data-gov-uk-2nd-line.html.md [hunk 1]
```diff
@@ -125,6 +125,18 @@ rails console
 >>> Sidekiq::Queue.new.each_with_object(Hash.new(0)) {|j, h| h[j.klass] += 1 }
 ```
 
+or on a single command line -
```
Comment (nicholsj) [inline source/manual/data-gov-uk-2nd-line.html.md:128]:
```suggestion
or on a single command line:
```

File: source/manual/data-gov-uk-2nd-line.html.md [hunk 2]
```diff
@@ -125,6 +125,18 @@ rails console
...
+
+```
+echo 'Sidekiq::Queue.all.map(&:size)' | k -n datagovuk exec -i deploy/datagovuk-publish -- rails c
+```
+
+If there are issues processing the sidekiq queue you can also clear it, this doesn't affect the operation of data.gov.uk -
```
Comment (nicholsj) [inline source/manual/data-gov-uk-2nd-line.html.md:134]:
```suggestion
If there are issues processing the Sidekiq queue you can clear it (this doesn't affect the operation of data.gov.uk):
```

**Review by nicholsj — APPROVED** [review]
A couple of suggested non-blocking tweaks. I haven't tested the commands though.

File: source/manual/data-gov-uk-2nd-line.html.md (hunk 1 above)
Comment (kentsanggds) [reply source/manual/data-gov-uk-2nd-line.html.md]: I've updated the text

File: source/manual/data-gov-uk-2nd-line.html.md (hunk 2 above)
Comment (kentsanggds) [reply source/manual/data-gov-uk-2nd-line.html.md]: I've updated the text

**Comment by MuriloDalRi** [comment]
@kentsanggds can we merge this?

**Pull request merged**